src/browser/squirrel-update.coffee

```diff
@@ -139,7 +139,7 @@ addCommandsToPath = (callback) ->
 
     atomShCommandPath = path.join(binFolder, 'atom')
     relativeAtomShPath = path.relative(binFolder, path.join(appFolder, 'resources', 'cli', 'atom.sh'))
-    atomShCommand = "#!/bin/sh\r\n\"$(dirname \"$0\")/#{relativeAtomShPath.replace(/\\/g, '/')}\" \"$@\""
+    atomShCommand = "#!/bin/sh\r\n\"$(dirname \"$0\")/#{relativeAtomShPath.replace(/\\/g, '/')}\" \"$@\"\r\necho"
```

Comment:
This is something I don't fully understand.  Using atom --wait from Powershell or cmd in Windows works using the cmd script.  Using as the git commit editor executes this Squirrel-generated sh script in Cygwin.  The script process doesn't return 0 and git won't accept the commit without the echo.  I tried exiting, returning the process return code and it didn't work unless there was an echo.  This makes the git commit work.